profiles/common/modules/features/cce_basic_config/cce_basic_config.install

```diff
@@ -2266,3 +2266,16 @@ function cce_basic_config_update_7214(&$sandbox) {
 
   $sandbox['#finished'] = empty($sandbox['max']) ? 1 : ($sandbox['progress'] / $sandbox['max']);
 }
+
+
+/**
+ * NEPT-182: Enable nexteuropa_trackedchanges module.
+ *
+ * Update enables the feature only if tmgmt_poetry is already installed.
+ */
+function cce_basic_config_update_7215() {
+  module_enable(array('nexteuropa_feature_set'));
+  if (module_exists('tmgmt_poetry')) {
+    module_enable(array('nexteuropa_dgt_connector'));
```

Comment:
Good remark. There is no requirements in the dgt connector nor the feature set module so that does not matter here.